fri/service_config.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+SERVICES=('init' 'ctl' 'pm' 'cleanup')
```

Comment:
This should be the methods you are defining.

So, it should be, 
SERVICES=('upload_file' 'execute')
and any method you introduce in main.py